internal/pkg/show/config/plugin/vault.go

```diff
@@ -0,0 +1,34 @@
+package plugin
+
+var VaultDefaultConfig = `tools:
+- name: vault
+  # name of the plugin
+  plugin: vault
+  options:
+    # need to create the namespace or not, default: false
+    create_namespace: true
+    repo:
```

Comment:
From an end user's perspective, it seems the "repo" information might be redundant. We should always try to use the official repo. So, maybe we can put some default values for this section and remove it from the config.